src/ContentValidationListenerFactory.php

```diff
@@ -10,12 +10,14 @@
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ContentValidationListenerFactory
+class ContentValidationListenerFactory implements FactoryInterface
```

Comment:
Without this implementation we get the error mentioned here : https://github.com/zfcampus/zf-content-validation/issues/72